### PR TITLE
Clean up unused vars and tasks, fix vagrant

### PR DESCRIPTION
**Pull request by bgw**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ Vagrant.configure("2") do |config|
   # Common Settings
   #
 
-  config.vm.hostname = "sovereign.local"
+  config.vm.hostname = "empress.local"
```
Comment (taoeffect) [inline Vagrantfile:8]: Depending on @al3x' response to #18, we might want to change this back to sovereign. Just keep that in mind.

File: vars/testing.yml [hunk 2]
```diff
@@ -5,78 +5,39 @@
 ###############################################################################
 
 # common
-domain: sovereign.local
-main_user_name: sovereign
-encfs_password: testPassword
+domain: empress.local
+main_user_name: empress
+timezone: America/Los_Angeles
```
Comment (taoeffect) [inline vars/testing.yml:10]: @PiPeep So why isn't the timezone being figured out from the system?

Comment (bgw) [inline vars/testing.yml:10]: The user's timezone is likely different from the server's original timezone. I think we should probably have this by default use the system's, and provide this shortcut for overwriting it.

**Comment by taoeffect** [comment]
Other than the weird timezone thing, looks good to merge, so i'll merge it.

**Pull request merged**